ReactAndroid/src/main/java/com/facebook/react/ReactInstanceManager.java

```diff
@@ -230,24 +229,19 @@ private static void initializeSoLoaderIfNecessary(Context applicationContext) {
    * {@link ReactRootView} is available and measured.
    */
   public void createReactContextInBackground() {
-    if (mUseDeveloperSupport) {
+    if (mUseDeveloperSupport && mJSMainModuleName != null) {
```

Comment:
Defaults to using the JSBundleLoader if no module name is provided. (Same effect as `forceJSBundleLoader`)